Add 16px transform-mirror-* icons and variants of format-segment-* icons

tesfabpel opened the pull request:
I have some doubts about the transform-mirror -horizontal and -vertical icons because the blue line in the center is not grid aligned but it is 1px wide...

This is the icon as it is in this pull-request:
![1px-wide](https://cloud.githubusercontent.com/assets/1371976/12376908/977f8e38-bd0a-11e5-8290-a4268de248e5.png)

This is the icon as it will be with the line set to 2px width:
![2px-wide](https://cloud.githubusercontent.com/assets/1371976/12376909/9c108524-bd0a-11e5-9b6d-732ea788acb5.png)

Which one do you think it is better?

Thanks

DarkknightAK commented:
the second screenshot is better with the 2px mirror line or you remove the mirror line in the 16px icon.

tesfabpel commented:
Hi,   
I've changed the look of those icons by removing the mirror line and changed the color of the right triangle.

It should be better now...

![1](https://cloud.githubusercontent.com/assets/1371976/12387294/cf6ee87c-bdcb-11e5-9454-691ef26a8f4d.png)

![2](https://cloud.githubusercontent.com/assets/1371976/12387299/d59d3bd6-bdcb-11e5-848c-daa172e62b9a.png)
